packages/teams-js/src/internal/chatConstants.ts

```diff
@@ -0,0 +1,4 @@
+export const deepLinkUrlPathForChat = '/l/chat/0/0';
```

Comment:
are these teams URL specific? If so it might be useful to name them or comment to that effect